### PR TITLE
numeric literal seperators: test change in CodePoint production

### PR DESCRIPTION
**Pull request by spectranaut**

### CODE REVIEW

File: test/language/literals/regexp/unicode-escape-nls-err.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+features: [numeric-separator-literal]
+negative:
+  phase: parse
+  type: SyntaxError
+---*/
+
+throw "Test262: This statement should not be evaluated.";
+
+/\u{1F_639}/u;
```
Comment (spectranaut) [inline test/language/literals/regexp/unicode-escape-nls-err.js]: this test fails when I run test harness for d8 -- but it does early error in chrome/firefox. Is there something obvious you see I'm missing, @leobalter ?

Comment (leobalter) [reply test/language/literals/regexp/unicode-escape-nls-err.js]: I don't think you're missing anything here, this test should be fine. Maybe you just caught an implementation bug?

cc @mathiasbynens

Comment (mathiasbynens) [reply test/language/literals/regexp/unicode-escape-nls-err.js]: V8 doesn’t throw *early* errors for invalid regular expression patterns. https://bugs.chromium.org/p/v8/issues/detail?id=896 It still throws at runtime, though.

Comment (leobalter) [reply test/language/literals/regexp/unicode-escape-nls-err.js]: Thanks for the clarification and showing the already known bug, @mathiasbynens!

**Pull request merged**